[DOC] Headless tutorial code causes error when trying to import icons

**Is this issue related to an existing part of the documentation? If so, please provide a link**
https://refine.dev/docs/tutorials/headless-tutorial/#adding-resources

**Describe the thing to improve**
After setting up tutorial as instructed (`npx superplate-cli -o refine-headless tutorial`), when trying to use icons as in the tutorial code
```javascript
import { PostIcon } from "icons";
```
I am getting error:
`Cannot find module 'icons' or its corresponding type declarations.ts(2307)`

Please note I did not execute `npm i @pankod/refine-simple-rest` as I had used `superplate` to bootstrap the project.

My system info:
MacOS `v12.6`
node `v16.17.0` 
nvm `v8.15.0`

From `package.json`:

```json
"dependencies": {
    "@pankod/refine-core": "^3.18.0",
    "@pankod/refine-react-router-v6": "^3.18.0",
    "@pankod/refine-react-table": "^4.7.2",
    "@pankod/refine-simple-rest": "^3.18.0",
    "@testing-library/jest-dom": "^5.12.0",
    "@testing-library/react": "^11.2.6",
    "@testing-library/user-event": "^12.8.3",
    "@types/jest": "^26.0.23",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-scripts": "^5.0.0",
    "web-vitals": "^1.1.1"
  },
  "devDependencies": {
    "@types/node": "^12.20.10",
    "@types/react": "^18.0.15",
    "@types/react-dom": "^18.0.6",
    "autoprefixer": "^10.4.12",
    "postcss": "^8.4.16",
    "tailwindcss": "^3.1.8",
    "typescript": "^4.7.4"
  },
```

**Describe the solution (optional)**
Update tutorial with missing package, maybe?

--- a/documentation/docs/tutorials/core.md
+++ b/documentation/docs/tutorials/core.md
@@ -86,6 +86,7 @@ First, run the **superplate** with the following command:
 ```
 npx superplate-cli -o refine-headless tutorial
 ```
+
 </TabItem>
 </Tabs>
 
@@ -285,7 +286,15 @@ export const App: React.FC = () => {
 
 The `icon` property of every single resource is can be used to display the resource in whatever way you want. For example in the sidebar or header. We'll use it when we'll create layout component.
 
-<details><summary>Show PostIcon</summary>
+<details>
+<summary>
+<strong>Show PostIcon</strong>
+<p style={{ marginBottom: 0 }}>
+<small>
+If you want to show the icon for your <code>posts</code> resource; you need to create an <code>icons.ts</code> file at <code>src</code> folder and add the following code to it. Its fully optional and you can just comment out the import and the <code>icon</code> property if you want to.
+</small>
+</p>
+</summary>
 <p>
 
 ```tsx title="icons.tsx"

